src/main/java/com/wolox/training/exceptions/IdMismatchException.java

```diff
@@ -1,12 +1,16 @@
 package com.wolox.training.exceptions;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
 /**
  * This class extends from RuntimeException and is used when two Id are not the same
  *
  * @author Gabriel Fernandez
  * @version 1.0
  */
-public class IdMismatchException extends RuntimeException{
+@ResponseStatus(code = HttpStatus.CONFLICT, reason = "The Object Id mismatch")
```

Comment:
The goal of the exception's messages is that whoever is using our API could get an idea of what went wrong. By receiving an error saying "the object id mismatch", I have no clue of which object caused that exception and it will be very difficult for me to correct it. You should specify if it was a **book** or a **user** that caused the id mismatch.